dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -1318,4 +1352,19 @@ protected void printAdminHelp() {
         out("");
     }
 
+
+
+    /**
+     * Reads a file whose format is either YAML or JSON and makes a JSON string out of the contents
+     * @param yamlRun
+     * @return
+     * @throws IOException, ParserException
+     */
+    public String fileToJSON(String yamlRun) throws IOException, ParserException {
```

Comment:
You should not have ParserException in the signature, as it is an unchecked exception. See [this Stackoverflow page](https://stackoverflow.com/questions/824217/should-methods-that-throw-runtimeexception-indicate-it-in-method-signature). You should keep it in the JavaDoc, and add a description, something like this:

```
@throws ParserException if the JSON or YAML is not syntactically valid
@throws IOException
```